exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -19,17 +22,18 @@ public class Game
         LinkedList<string> sportsQuestions = new LinkedList<string>();
         LinkedList<string> rockQuestions = new LinkedList<string>();
 
-        int currentPlayer = 0;
+        int currentPlayer;
         bool isGettingOutOfPenaltyBox;
 
-        public Game()
+        public Trivia_Game()
         {
-            for (int i = 0; i < 50; i++)
+            const int V = 50;
+            for (int Questionnumber = 0; Questionnumber < V; Questionnumber++)
             {
-                popQuestions.AddLast("Pop Question " + i);
-                scienceQuestions.AddLast(("Science Question " + i));
-                sportsQuestions.AddLast(("Sports Question " + i));
-                rockQuestions.AddLast(CreateRockQuestion(i));
+                popQuestions.AddLast("Pop Question " + Questionnumber);
+                scienceQuestions.AddLast(("Science Question " + Questionnumber));
+                sportsQuestions.AddLast(("Sports Question " + Questionnumber));
+                rockQuestions.AddLast(CreateRockQuestion(Questionnumber));
```

Comment:
'Sports - / Pop - / Science - / ' etc. should be stored in a variable / as a global property rather than hardcoded.